chrome/content/zotero/xpcom/connector/translator.js

```diff
@@ -303,7 +303,7 @@ Zotero.Translators = new function() {
 		}
 		
 		// Store
-		if(Zotero.isChrome || Zotero.isSafari) {
+		if((Zotero.isWebExtension || Zotero.isChrome) && !Zotero.isFx || Zotero.isSafari) {
```

Comment:
1) As noted elsewhere, I'm not sure why we need `isWebExtension` at all, and 2) I don't see how the `!Zotero.isFx` would ever be necessary, since `isFx` isn't true in the connector (at least currently — we can reconsider once we stop sharing code).

(Some fragile order of operations stuff here, in any case.)